webapp/src/components/gallery/gallery.tsx

```diff
@@ -25,6 +30,7 @@ type Props = {
 
 const Gallery = (props: Props): JSX.Element => {
     const {activeView, board, cards} = props
+    const hiddenCards = useAppSelector(GetHiddenCard)
```

Comment:
Reviewing the information flow for the `gallery`, `kanban` and `table` components, the `board`, `cards` and other pieces of the store are being fetched in the `CenterContent` component, so I think the `hiddenCards` should be fetched there as well, and passed through

Reply:
True, I will make the changes. Thanks